README.md

```diff
@@ -406,6 +406,47 @@ Loans include information from the item, including locations, holdingsRecordId a
 
 Requests include information from the item, including holdingsRecordId and instanceId.
 
+#### Expired Holds Report CSV
```

Comment:
Please can this section be expanded to include 
* a description of the logic used to construct the report
* the coupling this introduces with mod-circulation-storage by relying on the presence of the date to detect the transition

Reply:
updated